Resources/Prototypes/_Nuclear14/Recipes/Lathes/cooking.yml

```diff
@@ -89,15 +89,15 @@
     Plastic: 50
 
 - type: latheRecipe
-  id: N14FoodBowlBig
+  id: FoodBowlBig
```

Comment:
This is your linter error. This already exists in the upstream cooking.yml location. That's why ours was N14. The ID doesn't matter as long as its not a duplicate, what it produces is what matters.